src/FSharpPlus/Samples/Validations.fsx

```diff
@@ -130,4 +130,56 @@ module Email =
 
 
     let failureAll = email ""
-    // Failure [MustNotBeEmpty;MustContainAt;MustContainPeriod]
\ No newline at end of file
+    // Failure [MustNotBeEmpty;MustContainAt;MustContainPeriod]
+
+module MovieValidations=
+    type VError= | MustNotBeEmpty
+                 | MustBeAtLessThanChars of int
+                 | MustBeADate
+                 | MustBeOlderThan of int
+                 | MustBeWithingRange of int*int
+    module String=
+        let nonEmpty (x:string) : Validation<VError list,string> = 
+            if not <| String.IsNullOrEmpty x 
+            then Success <| x
+            else Failure [MustNotBeEmpty]
+        let mustBeLessThan (i:int) (x:string) : Validation<VError list,string> = 
+            if not <| isNull x && x.Length<=i
+            then Success <| x
+            else Failure [MustBeAtLessThanChars i]
+    module Number=
+        let inline mustBeWithin (from,to') (x)=
+            if from<= x && x <= to'
+            then Success <| x
+            else Failure [MustBeWithingRange ((int from),(int to'))]
+    module DateTime=
+        let classicMovie year (d:DateTime)=
+            if d.Year < year
+            then Success <| d
+            else Failure [MustBeOlderThan year]
+        let date (d:DateTime)=
+            if d.Hour = 0 && d.Minute =0 && d.Second = 0
+            then Success <| d
+            else Failure [MustBeADate]
+    type Genre=
+        |Classic
+        |PostClassic
+        |Modern
+        |PostModern
+        |Contemporary
+    type Movie = {
+        Id: int
+        Title: String
+        ReleaseDate: DateTime
+        Description: String
+        Price: decimal
+        Genre: Genre
+    }
+    with static member Mk(id,title,releaseDate,description,price,genre): Validation<VError list,Movie> =
+            fun title releaseDate description price->{ Id=id;Title=title;ReleaseDate=releaseDate;Description=description;Price=price;Genre=genre }
+            <!> ((String.nonEmpty title) <* (String.mustBeLessThan 100 title))
```

Comment:
I think the inner level of parens is not needed as the binary operator acts as separator.